vendor/capability-providers/blobstore-s3/tests/large_data.rs

```diff
@@ -28,7 +28,7 @@ fn gen_bytes(len: usize) -> (Vec<u8>, u32) {
 #[tokio::test]
 async fn run_all() {
     let opts = TestOptions::default();
-    env_logger::try_init().ok();
+    tracing::try_init().ok();
```

Comment:
Unfortunately, this won't work - you should use `tracing-subscriber` crate, like e.g. here https://github.com/wasmCloud/wasmCloud/blob/a457cb033cb25f0c29659224db553021244a4298/tests/wasmbus.rs#L462-L469 except we don't need the `,cranelift_codegen=warn,wasmcloud=trace` part

Please make sure to run `cargo test` from the root of this repository to check that it works